addons/website_sale/tests/test_ecommerce_access.py

```diff
@@ -40,3 +47,12 @@ def test_ecommerce_menu_visibility_public_user(self):
         self.menu.with_user(self.public_user).sudo()._compute_visible()
         # Check if menu is hidden for public user when ecommerce is restricted
         self.assertFalse(self.menu.is_visible)
+
+    def test_ecommerce_categories_visibility_public_users(self):
+        response = self.url_open('/shop')
+        self.assertEqual(response.status_code, 200)
+
+    def test_ecommerce_categories_visibility_portal_users(self):
+        self.authenticate(self.portal_user.login, self.portal_user.login)
+        response = self.url_open('/shop')
+        self.assertEqual(response.status_code, 200)
```

Comment:
The name of your test is misleading. I feel like you test if the customers can access to the page but the name suggests that you test if categories are visible on the page. 
```suggestion
        self.assertEqual(response.status_code, 200)  # Check that customers can access
```